README.rst

```diff
@@ -34,9 +34,15 @@ are omitted.
     permissions, with a command such as ``gpasswd --add $(id -u) docker &&
     newgrp``.
 
+``fixtures/drpm-unsigned``
+    The ``createrepo`` and ``makedeltarpm`` utilities must be available.
+
 ``fixtures/rpm``
     The ``createrepo`` and ``modifyrepo`` utilities must be available.
 
+``fixtures/rpm-unsigned``
+    The ``createrepo``, ``modifyrepo`` and ``rpm-sign`` utilities must be available.
```

Comment:
I was thinking whether I should put package name or utility name there.. Later I found it should be the utility name, and updated descriptions in .sh files. But I forgot to change the name in README as well :(